src/android/InAppBrowser.java

```diff
@@ -879,6 +910,48 @@ private void sendUpdate(JSONObject obj, boolean keepCallback, PluginResult.Statu
         }
     }
 
+    private void addTrustedCA() throws IOException, NoSuchAlgorithmException, CertificateException, KeyManagementException, KeyStoreException {
+        Log.d(LOG_TAG, "Adding trusted certificate if it exists");
```

Comment:
We use our own LOG, not the Android log.  This is causing compile errors in Travis CI.